App/Mixins/ListHelper.js

```diff
@@ -70,6 +74,18 @@ var ListHelper = {
     return this.username;
   },
 
+  getParseTitle: function() {
+    if (this.props.listProps.parse === true) {
+      return [
+        {type: 'url', style: styles.url, onPress: (url) => AppActions.launchExternalURL(url) },
+        {pattern: /@(\w+)/, style: styles.mention, onPress: (mention) => AppActions.launchRelativeItem(this.props.currentRoute, {replacePath: `follows/${mention.substring(1).toLowerCase()}/posts`}) },
```

Comment:
The user doesn't necessarily follow the person he mentioned. adding in `follows` will also make that the back button. I'd expect to have a change to the route parsing to allow something like `/bleonard/posts/jrlai`